### PR TITLE
fix(GraphQL): RHICOMPL-3059 Add debug code to check cached values

### PR DESCRIPTION
**Pull request by marleystipich2**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [x] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

This is debug code that will only be merged to stage so that we can get a little more details in the logs when we reproduce the 502 error for this ticket: https://issues.redhat.com/browse/RHICOMPL-3059
The logs don't give much info as of now.

### CODE REVIEW

File: app/models/profile.rb [hunk 1]
```diff
@@ -84,6 +84,13 @@ def supported_os_versions
   private
 
   def bm_versions
+    # rubocop:disable Rails/Output
+    if attributes['bm_versions']
+      puts '---Reached cached attributes in bm_versions---'
+    else
+      puts '---No cached attributes in bm_versions---'
```
Comment (skateman) [inline app/models/profile.rb:91]:
```suggestion
      `Rails.logger.info` '---No cached attributes in bm_versions---'
```

Comment (marleystipich2) [reply app/models/profile.rb:91]: Should this be like Rails.logger.info(...) or did you mean to add the `` around Rails.logger.info?

Comment (skateman) [reply app/models/profile.rb:91]: no `` :see_no_evil: typo

Comment (marleystipich2) [reply app/models/profile.rb:91]: Ok I figured 😄

**Comment by marleystipich2** [comment]
@skateman Does this look good now?

**Review by skateman — APPROVED** [review]
The Seal of Approval

**Pull request merged**